processing/src/main/java/org/apache/carbondata/processing/newflow/converter/impl/RowConverterImpl.java

```diff
@@ -158,8 +158,10 @@ public CarbonRow convert(CarbonRow row) throws CarbonDataLoadingException {
       if (!logHolder.isLogged() && logHolder.isBadRecordNotAdded()) {
         badRecordLogger.addBadRecordsToBuilder(copy.getData(), logHolder.getReason());
         if (badRecordLogger.isDataLoadFail()) {
-          String error = "Data load failed due to bad record: " + logHolder.getReason() +
```

Comment:
What is wrong with this message?

Reply:
Even when the BAD_RECORDS_ENABLE is set to true, the error message includes this: enable the bad record logger to know the detail reason.